### PR TITLE
Trash

### PR DESCRIPTION
**Pull request by Y2nn1c**
Trash

### CODE REVIEW

File: wiki/Community/en.md [hunk 1]
```diff
@@ -4,10 +4,11 @@ This page lists ways for people from all around osu! to get in touch with each o
 
 ## Places
 
+- [cavoe's osu! event](/wiki/Community/cavoe's_osu!_event)
 - [Community Discord servers](/wiki/Community/Discord_servers)
 - [Forum](/wiki/Community/Forum)
 - [Internet Relay Chat](/wiki/Community/Internet_Relay_Chat) (IRC)
-- [osu! Discord server](/wiki/Community/Discord_servers#official)
+- [Discord servers](/wiki/Community/Discord_servers#official)
```
Comment (minusQuantumNeko) [inline wiki/Community/en.md:11]: Link you are trying to rename is to official osu! Discord server, plus we already link community ones above (line 8)

```suggestion
- [osu! Discord server](/wiki/Community/Discord_servers#official)
```

**Review by minusQuantumNeko — CHANGES_REQUESTED** [review]

File: wiki/Community/en.md (hunk 1 above)
Comment (Y2nn1c) [reply wiki/Community/en.md:11]: Ok I didn't see the one above, however both links lead me to the same page, so should I just delete the "Community Discord servers" and stay with just "Discord servers"?

Comment (minusQuantumNeko) [reply wiki/Community/en.md:11]: @cl8n would like your opinion here

Comment (cl8n) [reply wiki/Community/en.md:11]: that looks like a leftover mistake from when the "osu discord server" article was turned into a section of "discord servers"

Comment (Walavouchey) [reply wiki/Community/en.md:11]: so like this

```suggestion
- [Discord servers](/wiki/Community/Discord_servers)
```

**Comment by Walavouchey** [comment]
badges aren't exclusive to tournaments, and user titles have almost nothing to do with them. the premise for the file moves here is off

**Comment by cl8n** [comment]
> however it doesn't make sense too keep them there, like how am I even properly gonna link them in Community.

not sure what you mean by this. Community is just an index page, you can add links to these wherever you want in that article, including in a new section if they don't fit in any of the existing ones.

**Comment by Y2nn1c** [comment]
> > however it doesn't make sense too keep them there, like how am I even properly gonna link them in Community.
> 
> not sure what you mean by this. Community is just an index page, you can add links to these wherever you want in that article, including in a new section if they don't fit in any of the existing ones.

Well but I have no clue on how to include them. The whole page is filled with ways on how people interact with eachother.
Both Badges and Titles don't really fit in their, especially when reading the introduction sentence at the top of the page

**Comment by Y2nn1c** [comment]
> badges aren't exclusive to tournaments, and user titles have almost nothing to do with them. the premise for the file moves here is off

So you think I should just keep em there?

**Pull request closed without merging**